config/services.yml

```diff
@@ -411,8 +411,12 @@ https://cmr.uat.earthdata.nasa.gov:
         env:
           <<: *default-argo-env
           STAGING_PATH: public/sds/swot-reproject
+    umm-s:
+      - S1237974711-EEDTEST
     collections:
-      - C1233860183-EEDTEST
+      # DO NOT UNCOMMENT THE FOLLOWING LINE - this collection is added via UMM-S and is left here
```

Comment:
I wonder if it'd be desirable to have our tests rely on a separate services.yml.

Reply:
I started doing that, but that meant making some extensive changes just to support testing. I also thought it might be good to have some mention in the services.yml that these collections don't need to be there. If anyone has a strong preference I could go the other route.

Reply:
That sounds like enough work to be its own ticket (if we did go that way).

Reply:
alternatively, you could mock `getServiceConfigs` to return whatever you want